### PR TITLE
Remove unnecessary tests, update Makefile.

### PR DESCRIPTION
**Pull request by vaikas**
Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

#### Ticket Link

Fixes

#### Release Note

```release-note

```

### CODE REVIEW

**Review by cpanato — APPROVED** [review]

File: test/e2e_test_insecure_registry.sh [hunk 1]
```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env bash
```
Comment (cpanato) [inline test/e2e_test_insecure_registry.sh:1]: oh, you removed this file but that is needed for the running test

**Review by cpanato — CHANGES_REQUESTED** [review]

File: .goreleaser.yml [hunk 2]
```diff
@@ -1,248 +0,0 @@
-project_name: cosign
```
Comment (cpanato) [inline .goreleaser.yml:1]: @vaikas are you planning to not use goreleaser anymore?

Comment (cpanato) [reply .goreleaser.yml:1]: discussed via slack :)

**Review by cpanato — APPROVED** [review]

File: test/e2e_test_insecure_registry.sh (hunk 1 above)
Comment (vaikas) [reply test/e2e_test_insecure_registry.sh:1]: Yeah, but it was not testing weh policy-controller, but instead cosign against insecure registry, so I updated the action to not run this.

**Pull request merged**